app/destination/page.js

```diff
@@ -1,54 +1,92 @@
 "use client";
 
-import { useState } from 'react';
+import { useState } from "react";
 
-import styles from '@/components/destination/destination.module.css';
-import { AddWishlistItem } from '@/components/destination/AddWishlistItem';
+import styles from "@/components/destination/destination.module.css";
+import { PlanetWishlistItem } from "@/components/destination/PlanetWishlistItem";
+import { AddWishlistItem } from "@/components/destination/AddWishlistItem";
 
-// TASK - React 1 week 2
-// Move this to its own file
-const PlanetWishlistItem = ({
+const PlanetCard = ({
   name,
-  onRemove,
+  description,
   thumbnail,
+  isSelected,
+  onAddOrRemovePlanet,
 }) => {
   return (
-    <div className={styles.wishlistItem}>
-      <img className={styles.wishlistItemThumbnail} src={thumbnail} alt="" />
-      <b>{name.toUpperCase()}</b>
-      <button onClick={onRemove}>remove</button>
+    <div className={styles.planetCard}>
+      <img className={styles.planetThumbnail} src={thumbnail} />
+      <div className={styles.planetDescription}>
+        <h2>
+          {name} {isSelected ? " - SELECTED" : ""}
+        </h2>
+        <p>{description}</p>
+      </div>
+      <button
+        className="roundButton"
+        onClick={() => onAddOrRemovePlanet(name, 0)}
+      >
+        {isSelected ? "REMOVE" : "ADD PLANET"}
+      </button>
     </div>
   );
-}
-
+};
 
 export const Destinations = () => {
   const [selectedPlanets, onAddPlanet] = useState([]);
```

Comment:
You could directly initialise the state with your array of planets

Reply:
Could you clarify the reasoning behind initializing the state with the array of planets?